spec/features/Welcome/quotes_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'Quotes Page' do
+  it "shows 0 results if no quotes were found" do
```

Comment:
In your expects, I would put exactly what appears on your page. So for example, when you expect to get zero results back:
```
expect(page).to have_content("0 total results found for dani")
```